common/src/main/java/org/apache/falcon/persistence/ExtensionMetadataBean.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.falcon.persistence;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+
+//SUSPEND CHECKSTYLE CHECK LineLengthCheck
+/**
+ * Table to store recipe metadata.
+ */
+
+@Table(name = "EXTENSION_METADATA")
+@Entity
+@NamedQueries({
+        @NamedQuery(name = PersistenceConstants.GET_ALL_RECIPES, query = "select OBJECT(a) from ExtensionMetadataBean a "),
+        @NamedQuery(name = PersistenceConstants.GET_RECIPE_LOCATION, query = "select a.location from ExtensionMetadataBean a where a.recipeName = :recipeName"),
+        @NamedQuery(name = PersistenceConstants.DELETE_ALL_RECIPES , query = "delete from ExtensionMetadataBean")
+})
+//RESUME CHECKSTYLE CHECK  LineLengthCheck
+public class ExtensionMetadataBean {
+    @Basic
+    @NotNull
+    @Id
+
+    @Column(name = "recipe_name")
+    private String recipeName;
+
+
+    @Basic
+    @NotNull
+    @Column(name = "extension_type")
+    private String extensionType;
+
+    @Basic
+    @NotNull
+    @Column(name = "description")
+    private String description;
```

Comment:
I believe one line description should be there. As we are using this field in the enumerate command.